services/galley/src/Galley/Data.hs

```diff
@@ -183,10 +184,23 @@ teamConversations t =
   map (uncurry newTeamConversation)
     <$> retry x1 (query Cql.selectTeamConvs (params Quorum (Identity t)))
 
-teamMembers :: forall m. (MonadThrow m, MonadClient m) => TeamId -> m [TeamMember]
-teamMembers t =
+teamMembers :: forall m. (MonadThrow m, MonadClient m) => TeamId -> Range 1 2000 Int32 -> m [TeamMember]
+teamMembers t limit =
   mapM newTeamMember'
-    =<< retry x1 (query Cql.selectTeamMembers (params Quorum (Identity t)))
+    =<< retry x1 (query Cql.selectTeamMembers (params Quorum (t, limit)))
+  where
+    newTeamMember' ::
```

Comment:
move this definition to the toplevel to avoid duplication?

Reply:
I was thinking about it, then I thought we want to kill one of them so its ok if it is not top level.